runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -815,7 +818,11 @@ func (p *Parser) parseMetricsView(node *Node) error {
 		return err
 	}
 
-	node.Refs = append(node.Refs, ResourceName{Name: table})
+	if table != "" {
+		node.Refs = append(node.Refs, ResourceName{Name: table})
+	} else {
+		node.Refs = append(node.Refs, ResourceName{Name: model, Kind: ResourceKindModel})
+	}
```

Comment:
```suggestion
	if tmp.Model != "" {
		// Not setting Kind because for backwards compatibility, it may actually be a source or an external table.
		node.Refs = append(node.Refs, ResourceName{Name: tmp.Model})
	}
```